### PR TITLE
cleanup(angular): deprecate and clean up angular schematics utilities

### PR DESCRIPTION
**Pull request by leosvelperez**
## Expected Behavior

Deprecate and clean up some Angular schematics utilities that are no longer going to be supported.

## Related Issue(s)

Fixes #

### CODE REVIEW

File: packages/angular/src/utils/testing.ts [hunk 1]
```diff
@@ -50,6 +53,9 @@ export function runMigration<SchemaOptions = any>(
     .toPromise();
 }
 
+/**
+ * @deprecated will be removed in v13.
+ */
 export function runExternalSchematic<SchemaOptions = any>(
```
Comment (Coly010) [inline packages/angular/src/utils/testing.ts:59]: What is the alternative?

Comment (leosvelperez) [reply packages/angular/src/utils/testing.ts:59]: None, as the PR description says, we're not going to support them moving forward, we support Nx DevKit utilities. These were always intended to be private anyway.

Comment (Coly010) [reply packages/angular/src/utils/testing.ts:59]: No problem 👍

**Review by Coly010 — APPROVED** [review]
LGTM! 🎉

**Pull request merged**